pbs/pbsrequest.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/spf13/viper"
 	"golang.org/x/net/publicsuffix"
 
+	"github.com/blang/semver"
```

Comment:
Probably should add this to glide.yaml

Reply:
oh man... yes. This is essential before merge